libs/audit-service/src/main/java/uk/gov/di/ipv/core/library/auditing/extension/AuditExtensionsVcEvidence.java

```diff
@@ -35,13 +41,29 @@ public AuditExtensionsVcEvidence(
             @JsonProperty(value = "iss", required = false) String iss,
             @JsonProperty(value = "evidence", required = false) String evidence,
             @JsonProperty(value = "successful", required = false) Boolean successful,
+            @JsonProperty(value = "vot", required = false) Vot vot,
             @JsonProperty(value = "isUkIssued", required = false) Boolean isUkIssued,
             @JsonProperty(value = "age", required = false) Integer age)
-            throws JsonProcessingException {
-        this.iss = iss;
-        this.evidence = evidence == null ? null : new ObjectMapper().readTree(evidence);
-        this.successful = successful;
-        this.isUkIssued = isUkIssued;
-        this.age = age;
+            throws AuditExtensionException {
+        try {
+            this.iss = iss;
+            this.evidence = evidence == null ? null : new ObjectMapper().readTree(evidence);
```

Comment:
Rather than creating a new ObjectMapper instance for every invocation of this constructor I think it would be more efficient to just create a static final field instance and reuse it.

Reply:
Not sure how you want this implemented. But I do agree that creating a new ObjectMapper instance every time the constructor is called can be inefficient. I have modified the code to use a static ObjectMapper instance instead that gets reused every time the constructor is called.

`private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();`

`this.evidence = evidence == null ? null : OBJECT_MAPPER.readTree(evidence);`